### PR TITLE
Add MOs for ERSPAN functionality

### PR DESCRIPTION
**Pull request by tanyatukade**
MOs to enable ERSPAN functionality.
- Creating VSPAN sessions using Source vspan and Destination vspan groups.

### CODE REVIEW

File: apicapi/apic_client.py [hunk 1]
```diff
@@ -335,6 +335,27 @@ class ManagedObjectClass(object):
                 'rsvswitchExporterPol-[uni/infra/vmmexporterpol-%s]'),
         'netflowVmmExporterPol': ManagedObjectName('infraInfra',
                                                    'vmmexporterpol-%s'),
+
+        # ERSPAN
+        'spanVSrcGrp': ManagedObjectName('infraInfra', 'vsrcgrp-%s'),
+        'spanVSrcGrp': ManagedObjectName('fvTenant', 'vsrcgrp-%s'),
```
Comment (tomflynn) [inline apicapi/apic_client.py:341]: Does this actually work? I see other similar cases use a different key. eg. fvRsCons vs fvRsCons__Ext when a class has multiple possible parents in the containment hierarchy

File: apicapi/apic_client.py [hunk 2]
```diff
@@ -335,6 +335,27 @@ class ManagedObjectClass(object):
...
+        'spanSpanLbl': ManagedObjectName('spanVSrcGrp', 'spanlbl-%s'),
+        'spanVSrc': ManagedObjectName('spanVSrcGrp', 'vsrc-%s'),
+        'spanRsSrcToVPort':
+        ManagedObjectName(
+                'spanVSrc',
+                'rssrcToVPort-[uni/tn-%(name)s/ap-%s/epg-%s/cep-%s]'),
+        'fvCEp': ManagedObjectName('fvAEPg', 'cep-%s'),
```
Comment (tbachman) [inline apicapi/apic_client.py:348]: After talking with Tom, I don't think we need the fvCEp MO. ERSPAN references an fvCEp, but we'll never need to configure it in OpenStack.

File: apicapi/apic_client.py [hunk 3]
```diff
@@ -335,6 +335,27 @@ class ManagedObjectClass(object):
...
+        'spanVDestGrp': ManagedObjectName('infraInfra', 'vdestgrp-%s'),
+        'spanVDestGrp': ManagedObjectName('fvTenant', 'vdestgrp-%s'),
```
Comment (tbachman) [inline apicapi/apic_client.py:350]: Same comment as Tom's on the spanVSrcGrp.

File: apicapi/apic_client.py (hunk 1 above)
Comment (tanyatukade) [reply apicapi/apic_client.py]: You are right, doubleunderscore is stripped off when handling these keys. Will address that in the latest commit

File: apicapi/apic_client.py (hunk 3 above)
Comment (tanyatukade) [reply apicapi/apic_client.py]: Addressed it in the latest commit

File: apicapi/apic_client.py (hunk 2 above)
Comment (tanyatukade) [reply apicapi/apic_client.py]: Removed it in the latest commit

**Comment by tbachman** [comment]
Please squash the commits.

**Pull request merged**